package.json

```diff
@@ -31,5 +31,10 @@
     "babel-cli": "~6.14.0",
     "babel-plugin-add-module-exports": "~0.2.1",
     "babel-preset-es2015": "~6.14.0"
+  },
+  "jspm": {
+    "main": "src/index.js",
```

Comment:
And, that's only my opinion, but you should not transpile any external libraries yourself.